third_party/silabs/efr32_sdk.gni

```diff
@@ -547,12 +547,14 @@ template("efr32_sdk") {
       ]
     }
 
-    if (chip_enable_multi_ota_encryption && chip_enable_multi_ota_requestor) {
-      defines += [ "SL_MATTER_ENABLE_OTA_ENCRYPTION=1" ]
```

Comment:
any specific reason to rename `SL_MATTER_ENABLE_OTA_ENCRYPTION`?

Reply:
done